Obsidian.API/Utilities/CommandHelpers.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Obsidian.API.Utilities;
+internal class CommandHelpers
+{
+    public const CommandIssuers DefaultIssuerScope = CommandIssuers.Any;
+
+    public const string DefaultPrefix = "/";
```

Comment:
Nice to have but the client always considers slash to be the default prefix

Keep it as-is though. Imagine in the future we get a different prefix, it'd be an easy swap. Doubt it though.